### PR TITLE
[change] Added autocomplete fields in OrganizatinOwner admin #325

### LINKED ISSUE #325: [change] Administrator group should not have permission to change organization owner
Non superadmins should not have permission to change the organization owner from the Organization change page, they should only have view permission.

It would be great if we can improve the readonly result of this part to show the name of the user, because right now in readonly mode the name of the org is shown instead.

### PR DESCRIPTION
**Pull request by pandafy**
Closes #325

### CODE REVIEW

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
I see this:

Screenshot from 2022-09-25 18-25-42

Screenshot from 2022-09-25 18-31-10

It doesn't look what we want right?

It must be because first and last name are empty, I think it may be better to show the username if first and last name are empty.

File: openwisp_users/base/models.py [hunk 1]
```diff
@@ -221,6 +221,19 @@ def clean(self):
...
+        """
+        Returns the connected user's full name or user's username if
+        full name method is unavailable (e.g. on a custom user class)
+        or if full name is an empty string.
+        """
+        try:
+            assert self.user.get_full_name() != ''
+            return self.user.get_full_name()
+        except (AssertionError, AttributeError):
+            return str(self.user.username)
```
Comment (nemesisdesign) [inline openwisp_users/base/models.py:235]: why not simply:

```python
return self.user.get_full_name() or str(self.user.username)
```

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

**Review by nemesisdesign — COMMENTED** [review]
PS: the latest solution works, it works also with the suggested simplification.

File: openwisp_users/base/models.py (hunk 1 above)
Comment (pandafy) [reply openwisp_users/base/models.py:235]: I adapted this code from django-organizations. I will update it.

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**